.github/workflows/test.yml

```
@@ -53,6 +57,8 @@ jobs:
       - run: "git push https://x-access-token:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git :test-create-new-pull-request"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - if: ${{ steps.run.outpus.result != 'created' || steps.run.outpus.pull-request-number > 0 }}
+        run: 'echo "Output of action is not as expected" && exit 1'
```

I think the right condition here would be

```suggestion
      - if: ${{ !(steps.run.outputs.result == 'created' && steps.run.outputs.pull-request-number > 0) }}
        run: 'echo "Output of action is not as expected" && exit 1'
```

Or split it up into two steps to make it easier to read

```suggestion
      - if: ${{ steps.run.outputs.result != 'created' }}
        run: 'echo "result output is  \"${{ steps.run.outputs.result }}\" but expected \"created\"" && exit 1'
      - if: ${{ steps.run.outputs.pull-request-number > 0 }}
        run: 'echo "pull-request-number output is  \"${{ steps.run.outputs.pull-request-number }}\" but expected a number" && exit 1'
```